### DISCUSSION TITLE
搭建后访问api会跳转到作者项目地址

### DISCUSSION THREAD

**Discussion by xmgmcat** [linweiyuan/go-chatgpt-api#186] [Q&A]
搭建后访问api会跳转到作者项目地址，ip:8080，求助什么原因

**Comment by linweiyuan** [comment]
当你直接访问这条链接的时候，其实心里是想返回什么

**Reply by xmgmcat** [reply]
不应该返回的是{"errorMessage":"Hello World."}，而不是跳转项目github地址吗

**Reply by linweiyuan** [reply]
最新的是跳转，不跳转是旧代码

**Reply by linweiyuan** [reply]
程序提示 Service is ready. 就是正常了，完全没有必要访问这条链接

用法不对是第一，用法不对还来提 issue 我就有点不明白

**Comment by linweiyuan** [comment]
https://github.com/linweiyuan/go-chatgpt-api/issues/173

**Reply by xmgmcat** [reply]
这个是正确的，但我现在的问题是不是返回消息，而是直接跳转到github里来了

**Comment by linweiyuan** [comment]
这不是说明 go-chatgpt-api 不能用，用法在文档里有

**Comment by xmgmcat** [comment]
我的问题，没了解清楚，感谢指教
